types/src/system_contract_errors/auction.rs

```diff
@@ -186,7 +195,11 @@ impl TryFrom<u8> for Error {
             d if d == Error::WithdrawDelegatorReward as u8 => Ok(Error::WithdrawDelegatorReward),
             d if d == Error::WithdrawValidatorReward as u8 => Ok(Error::WithdrawValidatorReward),
             d if d == Error::TransferToUnbondingPurse as u8 => Ok(Error::TransferToUnbondingPurse),
+
             d if d == Error::RecordEraInfo as u8 => Ok(Error::RecordEraInfo),
+            d if d == Error::CLValue as u8 => Ok(Error::CLValue),
+            d if d == Error::GasLimit as u8 => Ok(Error::GasLimit),
+            d if d == Error::InvalidContext as u8 => Ok(Error::InvalidContext),
```

Comment:
Could we move this line up so that it's in order?

Also, how did we miss this before?

Reply:
My bad it's actually duplicated branch and clippy didn't complain